### ISSUE TITLE
Why does Link automatically start to run when I load "Group the Loop" inside AUM and enable Link?

### ISSUE THREAD

**Issue by jmuheim** [Ableton/link#84]
Hello dear Ableton Link people

I'm trying to get my head around how Link exactly works, as I need to highly time-sensitively sync some MIDI actions when running AUM and GTL together; i'm using Mozaic to do the MIDI stuff.

At the time being, I don't really understand whether it is possible to influence the Start/Stop of Ableton Link. As it seems, Link is just running in the background and I don't have any influence on its running state, although I'd like to tell Link: Please start exactly **now**!

The following video demonstrates what I'm talking about: Ableton Link automatically runs when GTL connects to it:

https://youtu.be/s8jr7h9s_sk

Is this the expected behaviour? If so, why? And why isn't it possible to start/stop Ableton Link manually?

Thanks a lot for your effort.

**Comment by fgo-ableton** [comment]
You're right. The Link timeline is always running in the background. What you see in AUM is a visualization of the current phase of the Link timeline. When starting an app in a Link session, it will wait for the next downbeat to start playback.
Here's a video that explains this: https://www.youtube.com/watch?v=_Jzz6GKdcFA
